### PR TITLE
improve documentation: readme

### PR DESCRIPTION
**Pull request by fzzr-**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +11,21 @@
 <!-- TODO: describe project structure, architecture principles, etc.. -->
 
 
+## Related Repositories
+
+- [Dnode][] - Dfinance Blockchain node.
+- [PegZone][] - PegZone smart contracts.
+- [OracleApp][] - oracle node, that fetch price feed from exchanges.
```
Comment (damirka) [inline README.md:18]: which fetches price feeds

File: README.md [hunk 2]
```diff
@@ -154,6 +167,69 @@ To build your stdlib run:
 stdlib-builder /path-to-your/std-lib -po ./stdlib.json
 ```
 
+
+### Configuration actual for both
+
+#### Positional arguments:
+
+__DVM__ and __compiler__ requires second positional argument described as `<data-source address>`.
```
Comment (damirka) [inline README.md:175]: both require positional argument...

**Comment by fzzr-** [comment]
added documentation for #88

File: README.md [hunk 3]
```diff
@@ -154,6 +167,69 @@ To build your stdlib run:
...
+This is URI of a data source server, typically [Dnode][], local or external.
+This argument can be ommited because we'll read the `DVM_DATA_SOURCE` [environment variable][environment variables] as fallback.
+
+Positional arguments have higher priority than [environment variables][], so overrides their if specified.
```
Comment (damirka) [inline README.md:179]: meh.

, and override them when specified.

Comment (damirka) [reply README.md:179]: Same fix below.

File: README.md (hunk 1 above)
Comment (fzzr-) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (fzzr-) [reply README.md]: fixed, thanks

File: README.md (hunk 3 above)
Comment (fzzr-) [reply README.md]: fixed

Comment (fzzr-) [reply README.md]: fixed

**Pull request merged**